frontend/src/components/SearchByCriteria.js

```diff
@@ -23,9 +25,23 @@ function SearchByCriteria() {
     getInitData();
   }, []);
 
+  const searchByValues = async (values) => {
+    try {
+      const response = await axios.post('/api/recipes/search/criteria', values);
+      if (response.data !== null) {
+        setStatus('Sikeres keresés');
+        setRecipes(response.data);
+      }
+      setStatus('sikertelen keresés');
```

Comment:
Lehet, hogy ide kéne egy else, mert így mindenképp sikertelen keresés lesz a státusz :)

Reply:
A nap végén, így is úgy is, bele esik

Reply:
created else